convert/src/routers/text.py

```diff
@@ -0,0 +1,30 @@
+from typing import Dict
+
+from fastapi import APIRouter, status, HTTPException
+
+from src.config import minio_client
+from src.logger import get_logger
+from src.vertex_to_bd.models.text_model import TextRequest
+from src.vertex_to_bd.text_to_badgerdoc_use_case import TextToBDConvertUseCase
+
+router = APIRouter(prefix="/text", tags=["text"])
+LOGGER = get_logger(__file__)
```

Comment:
May you use logger as it is in other services. Here's an [example](https://github.com/epam/badgerdoc/blob/main/annotation/app/logger.py) from annotation

Reply:
Fixed